src/main/steps/applicant/yourdocuments/alldocuments/drug_alcohol_tests/content.ts

```diff
@@ -47,25 +46,12 @@ export const form: FormContent = {
 
 export const generateContent: TranslationFn = content => {
   const translations = languages[content.language]();
-  const orders: object[] = [];
-  for (const doc of content.userCase?.citizenUploadedDocumentList || []) {
-    if (
-      doc.value.isApplicant === content.byApplicant &&
-      doc.value.documentType === documents_list_items_en.drug_and_alcohol_tests
-    ) {
-      const uid = doc.value.citizenDocument.document_url.substring(
-        doc.value.citizenDocument.document_url.lastIndexOf('/') + 1
-      );
-      orders.push({
-        href: `${CITIZEN_DOWNLOAD_UPLOADED_DOCS}/${uid}`,
-        createdDate: doc.value.documentDetails.documentUploadedDate,
-        fileName: doc.value.citizenDocument.document_filename,
-      });
-    }
-  }
+  const uploadedBy = content.additionalData?.req.session.applicationSettings.docToView.uploadedBy;
```

Comment:
assign content.additionalData?.req.session.applicationSettings?.docToView to one constant variable and then access uploadedBy & docType. Apply this for all other content.ts